src/java/defrac/game/Game.java

```diff
@@ -144,15 +144,16 @@ private void gameOver() {
 
         state = GAME_OVER;
 
-        finish("Game over");
```

Comment:
Actually I don't think that the score is interesting at all. The games purpose is to get 2048.